docs/ruleset_functions/get_SHW_types_and_SHW_use.md

```diff
@@ -0,0 +1,34 @@
+
+## get_SHW_types_and_SHW_use
+
+Description: This function gets all the SHW Uses and the SHW space types and sorts them into a dictionary where the keys are the ServiceWaterHeatingSpaceOptions2019ASHRAE901 and values are a list if ServiceWaterHeatingUse.ids  
+
```

Comment:
general note on schema, ServiceWaterHeatingSpaceOptions2019ASHRAE901, these are building area types and are not typically called space types. Could cause potential confusion maybe. Maybe we should modify ServiceWaterHeatingBuildingAreaOptions2019ASHRAE901